features/step_definitions/create_articles_steps.rb

```diff
@@ -2,6 +2,10 @@
   visit root_path
 end
 
+Given("show me the page") do
+  save_and_open_page
+end
+
```

Comment:
we can delete this, and the the feature test for it

Reply:
The feature file don't have it and we can keeps this around for the future :)